server/src/test/scala/de/frosner/broccoli/templates/SignalRefreshedTemplateSourceSpec.scala

```diff
@@ -8,8 +9,9 @@ import sun.misc.Signal
 class SignalRefreshedTemplateSourceSpec extends Specification with Mockito {
   "Receiving a SIGUSR2 signal" should {
     "update the cache" in {
+      val signalManager = new UnixSignalManager()
```

Comment:
Now that the signal manager moved out of the cached template source, please rewrite this test with a mocked signal manager so that it doesn't need the actual USR2 signal anymore.